test/Riok.Mapperly.Tests/Mapping/DateTimeTest.cs

```diff
@@ -1,18 +0,0 @@
-namespace Riok.Mapperly.Tests.Mapping;
-
-public class DateTimeTest
-{
-    [Fact]
-    public void DateTimeToDateOnly()
```

Comment:
I think it makes sense to keep this test to ensure backward compatibility, just move it to the new test class.